wildfly/connectors/file/feature-pack-file/src/main/resources/modules/system/layers/dv/org/jboss/teiid/resource-adapter/s3/main/module.xml

```diff
@@ -25,5 +24,6 @@
         <module name="com.fasterxml.jackson.core.jackson-annotations"/>
         <module name="org.apache.httpcomponents.core"/>
         <module name="org.joda.time"/>
+        <module name="com.amazonaws" />
```

Comment:
Why this change?
Like in module.xml from s3?

Reply:
The module for com.amazonaws:aws-java-sdk-core is called com.amazonaws (that was the existing module name, but it would be made clearer).  And yes it is used by both simpledb and s3.